lib/routes/eprice/rss.js

```diff
@@ -0,0 +1,70 @@
+const got = require('@/utils/got');
+const parser = require('@/utils/rss-parser');
+const cheerio = require('cheerio');
+const { parseDate } = require('@/utils/parse-date');
+
+module.exports = async (ctx) => {
+    const region = ctx.params.region || 'tw';
+
+    const feed = await parser.parseURL(`https://www.eprice.com.${region}/news/rss.xml`);
+
+    const items = await Promise.all(
+        feed.items.map(async (item) => {
+            item.link = item.link.replace(/^http:\/\//i, 'https://');
+            const cache = await ctx.cache.get(item.link);
```

Comment:
tryGet

Reply:
Resolved in https://github.com/DIYgod/RSSHub/pull/8118/commits/551e57edc0f4d9bbec8632fe91a9297e7ec396dc